lib/manageiq/environment.rb

```diff
@@ -16,7 +16,6 @@ def self.manageiq_plugin_update(plugin_root = nil, force_bundle_update: true)
       # determine plugin root dir. Assume we are called from a 'bin/' script in the plugin root
       plugin_root ||= Pathname.new(caller_locations.last.absolute_path).dirname.parent
 
-      setup_gemfile_lock if ENV["CI"]
```

Comment:
At this point the Gemfile.lock will have already been patched and this would just overwrite it.

Reply:
I think we still need this for build...or does build manually copy the file?

Reply:
The build doesn't call `bin/setup` in each plugin dir, it only does this for the core repo

Reply:
Oh ok - I forgot that core doesn't call into these methods...I _really_ need to push my branch where it does, cause I keep confusing myself haha.

Reply:
Ok. So is this line needed or not?